cmd/edit/cluster/cmd.go

```diff
@@ -966,3 +962,13 @@ func auditLogInteractivePrompt(r *rosa.Runtime, cmd *cobra.Command, cluster *cmv
 	}
 	return
 }
+
+func PromptUserToAcceptRegistryChange(r *rosa.Runtime) bool {
+	prompt := "Changing any registry related parameter will trigger a rollout across all machinepools " +
+		"(all machinepool nodes will be recreated, following pod draining from each node). Do you want to proceed?"
+	if !confirm.ConfirmRaw(prompt) {
+		r.Reporter.Warnf("You have not changed any registry configuration.")
```

Comment:
```suggestion
		r.Reporter.Infof("No changes to registry configuration.")
```

Reply:
Thanks @andreadecorte , updated the text